spec/features/update_username_spec.rb

```diff
@@ -12,12 +12,12 @@
   context 'Update Username page' do
     it 'visits and enter updated username' do
       VCR.use_cassette('update_username') do
-        create(:user, id: 1, username: 'Old Username', global_id: '14093230')
+        create(:user, username: 'Old Username', global_id: '14093230')
```

Comment:
Include a comment noting that this global ID belongs to user 'Ragesock', since that isn't obvious just from reading this file.

Reply:
Okie